antora.yml

```
@@ -14,9 +12,8 @@ asciidoc:
     # We try to fetch the latest from GitHub at build time
     # --
     full-version: 24.2.1
-    redpanda-beta-version: 24.2.1-rc5
-    latest-release-commit: 'b5ade3f40'
-    latest-operator-version: 'v2.1.11-24.1.1'
+    latest-release-commit: '08ae9f9c'
+    latest-operator-version: 'v2.1.11-24.2.1'
```

```suggestion
    latest-operator-version: 'v2.1.11-24.1.1'
```

This won't be released until after 24.2 gos live. We have an automation that fetches the latest version as long as it's greater than the version defined here. So best to leave this for now to avoid displaying an unreleased version.